### ISSUE TITLE
Javascript: Image Carousel project requires some clarification

### ISSUE THREAD

**Issue by hbar1st** [TheOdinProject/curriculum#29123]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the section of the JS curriculum which requests the creation of an image carousel here <https://www.theodinproject.com/lessons/node-path-javascript-dynamic-user-interface-interactions>, the instructions never specifically say which method they would like us to use to achieve this. 

For eg. The image carousel can be setup to use internal links to traverse between the images, or another method is to use scrollToView to move the images into position or finally a method to use left property values to manually move the slider div can be used. (here are two different implementations I did: Using a combo of direct internal links and scrollToView calls, I created this demo <https://hbar1st.github.io/demo-poster-slider/>, while I changed the code later on in a different branch to use the left property to move the slider here (sorry no demo) <https://github.com/hbar1st/demo-poster-slider/tree/position>

I think if there is an intention that one method should be favored over another, the step can be clearer about that in its description as it currently doesn't really suggest a specific method or explicitly require one. I personally found the scrollToView method to be better than changing the left position (no maths, and code is easier to read plus the overall effect looks like scrolling for free without any use of animation or translation).

Just some thoughts on this exercise which will hopefully lead to either adding a hint to suggest something to the learners or being more specific that any method to move to the images will be acceptable. Thanks!

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-dynamic-user-interface-interactions

**Comment by MaoShizhong** [comment]
Agree the wording could do with clarification. It seems like it's alluding towards absolute positioning and manipulating `left`, which is also what's been taught in the previous course, but since it just says "positioning" generally and not something like "`position`ing" (or something more explicit), it's not clear if it expects a `position`-based approach or if that's actually just a generic "position the div via whatever method, including `scrollIntoView` if you happen to know about it".

**Comment by wise-king-sullyman** [comment]
I think some clarity would probably be good as well. @TheOdinProject/javascript anyone disagree?

**Comment by bycdiaz** [comment]
I like the idea of adding clarification but I wonder if there's some value in making learners aware of the possibilities and invite them to weigh the options and make a call? Feels a little funny to me to just say "do it this way".

@hbar1st thanks for raising this!

**Comment by mao-sz** [comment]
I think over-specifying wouldn't be as sensible. Absolute positioning is probably going to be the more intuitive approach, I think, but if someone finds an alternative that still meets the general requirements, all power to them.

## Acceptance criteria

- Add the following sentence to the end of step 2 for the image carousel:
  > How you approach this is up to you.

Please comment below if you would like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by frrst-ian** [comment]
Can you assign this to me? Thanks.

**Comment by mao-sz** [comment]
All yours @frrst-ian
